### PR TITLE
Fabo/Fix Polkadot balances

### PR DESCRIPTION
**Pull request by faboweb**
The balances where not showing correctly

### CODE REVIEW

File: api/lib/subscription/slashing.js [hunk 1]
```diff
@@ -74,8 +74,9 @@ class SlashingMonitor {
     )
 
     // requires some more logic to not spam the notifications if a validator is down for 1000 blocks
+    const minMissedBlocks = this.networkId === "kava-testnet" || this.networkId === "kava-mainnet" ? 5 : 1
```
Comment (Bitcoinera) [inline api/lib/subscription/slashing.js:77]: Nice!

File: api/lib/source/polkadotV0-source.js [hunk 2]
```diff
@@ -198,10 +198,9 @@ class polkadotAPI {
   async getBalancesV2FromAddress(address, fiatCurrency) {
     const api = await this.getAPI()
     const account = await api.query.system.account(address)
-    const totalBalance = account.data.free
-    const freeBalance = BigNumber(totalBalance.toString()).minus(
-      account.data.miscFrozen.toString()
-    )
+    const { free, feeFrozen } = account.data.toJSON()
+    const totalBalance = BigNumber(free)
+    const freeBalance = BigNumber(free).minus(feeFrozen)
```
Comment (Bitcoinera) [inline api/lib/source/polkadotV0-source.js:203]: Does this fix our current UX problem with balances not changing after a delegation?

Comment (faboweb) [reply api/lib/source/polkadotV0-source.js:203]: I think though

Comment (jbibla) [reply api/lib/source/polkadotV0-source.js:203]: what is `feeFrozen`? it is not obvious from the name.

Comment (faboweb) [reply api/lib/source/polkadotV0-source.js:203]: Not obvious toe either. It's from the Polkadot API. It seems to hold the locked values. Need to monitor further.

Comment (Bitcoinera) [reply api/lib/source/polkadotV0-source.js:203]: It is not working on my side... I just get reduced fees, like before (trying with Westend)

Comment (Bitcoinera) [reply api/lib/source/polkadotV0-source.js:203]: Yes, I can confirm it is not working... not sure yet what `feeFrozen` stores but it is not the whole balance you aim to stake

Comment (Bitcoinera) [reply api/lib/source/polkadotV0-source.js:203]: yes, it works! :tada: sorry for the confusion :laughing:

**Pull request merged**